repo_tests/no_commit_trailers_anchor_test.py

```
@@ -0,0 +1,111 @@
+    """
+    text = WORKFLOW.read_text(encoding="utf-8")
+    match = re.search(r"^\s*agent_re='([^']+)'", text, re.MULTILINE)
+    assert match, "agent_re not found in the workflow -- this test cannot see what it checks"
+    return match.group(1)
+
+
+def _matches(pattern: str, line: str) -> bool:
+    """Ask grep, not Python: the workflow runs `grep -qE`, and the dialects differ."""
+    return (
+        # Fixed argv; inputs are module constants.
+        subprocess.run(  # nosec B603 B607
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)